rdtools/normalization.py

```diff
@@ -567,7 +551,7 @@ def energy_from_power(power, target_frequency=None, max_timedelta=None):
     return energy
 
 
-def trapz_aggregate(time_series, target_frequency, max_timedelta=None):
+def _aggregate(time_series, target_frequency, max_timedelta=None, series_type='right_labeled'):
     '''
     Returns a right-labeled series with frequency target_frequency generated by
     aggregating `time_series` with the trapezoidal rule (in units of hours).
```

Comment:
Docstring updates: update this line so it says the method depends on `series_type`, also add `series_type` to the parameter description list

Reply:
Also need to drop `trapz_aggregate` from `api.rst`